frontend/webpack.config.js

```diff
@@ -24,14 +25,14 @@ const version = versionMatches[1];
 const targets = {
   browsers: ['last 2 versions']
 };
-const dependencies = require('./package.json').dependencies;
+const { dependencies } = require('./package.json');
+
 delete dependencies['open-iconic'];
 
 module.exports = {
   entry: {
     chainerui: [
       'babel-polyfill',
-      'whatwg-fetch',
```

Comment:
Can we really remove `whatwg-fetch` ?
If my memory is correct, we couldn't run chainerui on Edge or IE without `whatwg-fetch` .

Reply:
OK, I will revert the change.